### PR TITLE
Replace = with ==

### PR DESCRIPTION
**Pull request by trodge**
If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Review by hao-nan-li — COMMENTED** [review]
Thanks for the catch, looks like there's another issue introduced by `TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute`. It would be safer to revert https://github.com/GoogleCloudPlatform/magic-modules/pull/8324/files and fix the error there? What do you think?

**Comment by trodge** [comment]
> Thanks for the catch, looks like there's another issue introduced by `TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute`. It would be safer to revert https://github.com/GoogleCloudPlatform/magic-modules/pull/8324/files and fix the error there? What do you think?

Should be fixed now.

**Comment by hao-nan-li** [comment]
> > Thanks for the catch, looks like there's another issue introduced by `TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute`. It would be safer to revert https://github.com/GoogleCloudPlatform/magic-modules/pull/8324/files and fix the error there? What do you think?
> 
> Should be fixed now.

Looks like the added test failed, @trodge are you planning to fix it, or shall we try to fix it in the original PR by reverting it?

File: mmv1/third_party/terraform/tests/resource_compute_disk_test.go.erb [hunk 1]
```diff
@@ -552,17 +552,18 @@ func TestAccComputeDisk_encryptionKMS(t *testing.T) {
...
+
+<% unless version == 'ga' -%>
 func TestAccComputeDisk_pdHyperDiskEnableConfidentialCompute(t *testing.T) {
 	t.Parallel()
 
-
 	context := map[string]interface{}{
 		"random_suffix":        RandString(t, 10),
-		"kms":                  BootstrapKMSKey(t),
+		"kms":                  BootstrapKMSKey(t).CryptoKey.Name,
```
Comment (hao-nan-li) [inline mmv1/third_party/terraform/tests/resource_compute_disk_test.go.erb:562]: I've seen in other test file they bootstrap the cryptoKey by putting `acctest.BootstrapKMSKey(t).CryptoKey.Name`, perhaps that could fix the test?

**Comment by trodge** [comment]
I've merged in @SarahFrench's PR and skipped the test for now because the required disk type doesn't exist in the test zone.

**Comment by hao-nan-li** [comment]
What's the plan here, shall we still merge this PR for the fix?

**Comment by trodge** [comment]
> What's the plan here, shall we still merge this PR for the fix?

Yes, we still want to merge this because other tests are being hidden by the assignment of `ga` to `version`.

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**